src/main/java/seedu/clinic/MainApp.java

```diff
@@ -69,29 +74,50 @@ public void init() throws Exception {
     }
 
     /**
-     * Returns a {@code ModelManager} with the data from {@code storage}'s clinic and {@code userPrefs}. <br>
+     * Returns a {@code ModelManager} with the data from {@code storage}'s clinic, user macros, and
```

Comment:
Will there be a {@code userMacros}? I'm not too sure about how this component of the code.

Reply:
Actually both clinic and user macros are part of the {@code storage} thats why they are not in the brackets.

Reply:
Oh alright! 👍